pkg/chains/formats/intotoite6/testdata/taskrun1.json

```diff
@@ -7,11 +7,11 @@
             },
             {
                 "name": "CHAINS-GIT_COMMIT",
-                "value": "abcd"
+                "value": "sha:taskrun"
             },
             {
                 "name": "CHAINS-GIT_URL",
-                "value": "https://git.test.com"
```

Comment:
Silly question, why change this to git1?

Reply:
Aha, I just wanted to differentiate it with the value defined in default. Changed this back and named value in default `defaultgit.test.com` haha.